sycl/source/detail/pi.cpp

```diff
@@ -439,8 +446,6 @@ static void initializePlugins(std::vector<plugin> &Plugins) {
   }
 
 #ifdef XPTI_ENABLE_INSTRUMENTATION
-  GlobalHandler::instance().getXPTIRegistry().initializeFrameworkOnce();
```

Comment:
Could you please tell why this line is being removed?

Reply:
It's an unwanted change.
Reverted it.